test/runtime_test.dart

```diff
@@ -50,26 +48,52 @@ void main() {
   test.group('All example programs run without errors', () {
     Directory directory = Directory('./examples/');
     directory.listSync().forEach((file) {
```

Comment:
A better approach that would support any kind of nested directories is to use `directory.listSync(recursive: true)`, then most of the logic inside to relist the contents of subdirectories won't be needed